petclinic/actors/customers/actor.mk

```diff
@@ -57,6 +57,7 @@ all:: $(DIST_WASM)
 $(DIST_WASM): $(UNSIGNED_WASM) Makefile
```

Comment:
you shouldn't need an actor.mk in this folder. From your main folder, 
`include ../../../../build/makesilfes/actor.mk`
(with the appropriate number of `../`)